.github/workflows/wbstack.sync.trigger.yml

```diff
@@ -7,6 +7,11 @@ on:
         description: 'The branch name to run against'
         required: true
         default: 'not-main'
+      composer:
+        description: 'Also do a composer update (true/false)'
+        type: boolean
+        required: true
+        default: true
```

Comment:
this should surely default false to avoid changing behaviour

Reply:
Sounds good!